chainercv/links/model/mobilenet/mobilenet_v2.py

```diff
@@ -0,0 +1,249 @@
+    automatically initializes all the parameters with it.
+    When a string in the prespecified set is provided, a pretrained model is
+    loaded from weights distributed on the Internet.
+    The list of pretrained models supported are as follows:
+
+    * :obj:`imagenet`: Loads weights trained with ImageNet. \
+        When :obj:`arch=='tf'`, the weights distributed \
+        at tensorflow/models
+        `<https://github.com/tensorflow/models/tree/master/research/slim/nets/mobilenet>`_ \ # NOQA
+        are used.
+
+    Args:
+        n_class (int): The number of classes. If :obj:`None`,
+            the default values are used.
+            If a supported pretrained model is used,
+            the number of classes used to train the pretrained model
+            is used. Otherwise, the number of classes in ILSVRC 2012 dataset
+            is used.
+        pretrained_model (string): The destination of the pre-trained
+            chainer model serialized as a :obj:`.npz` file.
+            If this is one of the strings described
+            above, it automatically loads weights stored under a directory
+            :obj:`$CHAINER_DATASET_ROOT/pfnet/chainercv/models/`,
+            where :obj:`$CHAINER_DATASET_ROOT` is set as
+            :obj:`$HOME/.chainer/dataset` unless you specify another value
+            by modifying the environment variable.
+        mean (numpy.ndarray): A mean value. If :obj:`None`,
+            the default values are used.
+            If a supported pretrained model is used,
+            the mean value used to train the pretrained model is used.
+            Otherwise, the mean value used by TF's implementation is used.
+        initialW (callable): Initializer for the weights.
+        initial_bias (callable): Initializer for the biases.
+
+    """
+    # Batch normalization replicating default tensorflow slim parameters
+    # as used in the original tensorflow implementation.
+    _bn_tf_default_params = {
+        "decay": 0.999,
+        "eps": 0.001,
+        "dtype": chainer.config.dtype
+    }
+
+    _models = {
+        'tf': {
+            1.0: {
+                'imagenet': {
+                    'param': {
+                        'n_class': 1001,  # first element is background
+                        'mean': _tf_mobilenetv2_mean,
+                        'scale': _tf_mobilenetv2_scale,
+                    },
+                    'overwritable': ('mean', 'scale'),
+                    'url':
+                    'https://chainercv-models.preferred.jp/mobilenet_v2_depth_multiplier_1.0_imagenet_converted_2019_05_13.npz',  # NOQA
+                }
+            },
+            1.4: {
+                'imagenet': {
+                    'param': {
+                        'n_class': 1001,  # first element is background
+                        'mean': _tf_mobilenetv2_mean,
+                        'scale': _tf_mobilenetv2_scale,
```

Comment:
Can we rescale input value for TF pretraiend model to `[0, 255]` instead of `[0, 1]`?
You can do it by scaling the `W` of the first conv layer when converting pretrained model.
This simplifies the implementation because we no longer need `scale` option elsewhere.